### ISSUE TITLE
Missing BalanceTracker contract documentation

### ISSUE THREAD

**Issue by liamaharon** [hubiinetwork/nahmii-contracts#287]
I can't find documentation for the new BalanceTracker contract under `docs/`

**Comment by jijordre** [comment]
Right. I am scanning the market for document generator tools that are up to date w.r.t. recent additions to Solidity. The ones I have tried are unfortunately outdated and thus either fail to complete or generate incomplete documentation. I attempted a patch of [soldoc](https://github.com/tsuberim/soldoc) but it sadly requires more work. Will keep you posted, mate.

**Comment by liamaharon** [comment]
Not a high priority, don't stress :)
